library/std/src/lib.rs

```diff
@@ -219,22 +219,22 @@
 // std is implemented with unstable features, many of which are internal
 // compiler details that will never be stable
 // NB: the following list is sorted to minimize merge conflicts.
-#![feature(alloc_error_handler)]
-#![feature(alloc_layout_extra)]
 #![feature(allocator_api)]
 #![feature(allocator_internals)]
+#![feature(alloc_error_handler)]
+#![feature(alloc_layout_extra)]
```

Comment:
I don't necessarily mind sorting the features here (though it might cause unnecessary merge conflicts), but these ones specifically were already sorted correctly. The `_` comes before `a`.

Reply:
Ah, probably a locale thing:

```
$ LANG=C sort thing
#![feature(alloc_error_handler)]
#![feature(alloc_layout_extra)]
#![feature(allocator_api)]
#![feature(allocator_internals)]
```
```
$ LANG=en_GB.UTF-8 sort thing
#![feature(allocator_api)]
#![feature(allocator_internals)]
#![feature(alloc_error_handler)]
#![feature(alloc_layout_extra)]
```

`rustfmt` uses the first one for imports though, so let's follow that one.

Reply:
:pensive: